app/src/androidTest/java/com/github/multimatum_team/multimatum/MainActivityTest.kt

```diff
@@ -116,6 +116,17 @@ class MainActivityTest {
         )
     }
 
+    @Test
+    fun goToAddDeadlineActivity() {
+        onView(withId(R.id.main_go_to_add_deadline)).perform(ViewActions.click())
+        Intents.intended(
+            allOf(
+                hasComponent(AddDeadlineActivity::class.java.name),
+                toPackage("com.github.multimatum_team.multimatum")
+            )
```

Comment:
the allOf() function is not necessary you can just do Intents.intended(hasComponent(AddDedlineActivity::class.java.name)
The best will be clearer and won't have useless line

Reply:
But this will remove the check of the correct package by doing so I think